tremor-script/eqc/spec.erl

```diff
@@ -164,6 +162,44 @@ spec_uop_int(S, N) ->
     ?SHRINK({oneof(['+', '-']), spec_inner_int(S, N - 1)},
 	    [spec_inner_int(S, N - 1)]).
 
+spec_uop_float(S, N) when N =< 1 ->
+    ?SHRINK({oneof(['+', '-']), float_or_float_local(S)},
+	    [float_or_float_local(S)]);
+spec_uop_float(S, N) ->
+    ?SHRINK({oneof(['+', '-']), spec_inner_float(S, N - 1)},
+	    [spec_inner_float(S, N - 1)]).
+
+patch_operation(S, N) ->
+    frequency([{1,
+		{insert, spec_inner_string(S, N - 1),
+		 spec_inner_no_float(S, N - 1)}},
+	       {1,
+		{upsert, spec_inner_string(S, N - 1),
+		 spec_inner_no_float(S, N - 1)}},
+	       {1,
```

Comment:
`patch_operation` does not cover all specified patch operations

Reply:
Yeah, I missed some 😅. Updated and ran tests again, thanks